src/Xamarin.Android.Tools.Bytecode/JavaDocumentScraper.cs

```diff
@@ -290,6 +290,45 @@ public static void LoadXml (String filename)
 				Log.Error ("Annotations parser error: " + ex);
 			}
 		}
+
+		public static JavaDocletType GetDocletType (string path)
+		{
+			var kind = JavaDocletType.DroidDoc;
+			char[] buf = new char[500];
+
+			string packagesHtml = Path.Combine (path, "packages.html");
+			if (File.Exists (packagesHtml) && File.ReadAllText (packagesHtml).Contains ("<body class=\"gc-documentation develop reference api "))
+				kind = JavaDocletType.DroidDoc2;
+
+			string indexHtml = Path.Combine (path, "index.html");
+			if (File.Exists (indexHtml))
+			{
```

Comment:
You have some curious formatting settings applied. You have space ` ` before `(` (yay), but `{` is on a new line (?).

That's...unexpectedly bizarre.

Please place the `{` on the same line as the `if`. :-)